js/ui/gantt/ui.gantt.js

```diff
@@ -118,10 +177,16 @@ class Gantt extends Widget {
         return GANTT_DEFAULT_ROW_HEIGHT;
     }
 
-    _updateWidth() {
-        const treeListWidth = this.option("treeListWidth");
+    _updateWidth(treeListWidth) {
         this._$treeListWrapper.width(treeListWidth);
         this._$treeList.width(treeListWidth);
+        this._$splitter.css('left', treeListWidth);
+        this._ganttView && this._ganttView._setWidth(this.$element().width() - treeListWidth - GANTT_SPLITTER_BORDER_WIDTH);
+    }
+
+    _dispose() {
```

Comment:
```suggestion
    _clean() {
```
`instance.repaint()` => `instance._clean()` + `instance._render()`
`instance.dispose()` => `instance._dispose()` (also calls the `_clean` method)